.codeclimate.yml

```diff
@@ -0,0 +1,14 @@
+version: "2"
+checks:
+  file-lines:
+    enabled: false
```

Comment:
why are this false??

Reply:
We agreed to disable some of the checks made by code climate. Are you suggesting that I place it as true on this particular configuration?

Reply:
Why did you decide to disable the checks rather than fixing them?

Reply:
Some of the checks don't apply for the react project because it is also flagging similar blocks of code in test which obviously should remain similar

Reply:
Which checks don't apply in React?

Reply:
I am raising this because your teammate has also put a comment to enable the same thing and the comments have not been resolved yet.

Reply:
Just resolved the issue

Reply:
You didn't answer my second [question](https://github.com/andela/ah-code-titans-frontend/pull/38#discussion_r255002650) kindly do?

Reply:
We have already discussed what we want to keep and remove. The current codeclimate config has been agreed on. @justmesam

Reply:
I wasn't requesting for you to change the values, I wanted to get the context and the reason why it was disabled also I noticed your teammate requested change but his comments were not resolved.
All you had to do was just give your honest answer on why you did your way and explain why the other comments were not resolved. Its as simple as that. 🤷‍♂️

Reply:
Sorry I did not explain the context well. We had agreed to change a few of the configs and leave others. I forgot to resolve the conversations.